qkrehdwns/likelion-assignment-crud/src/main/java/likelion/likelionassignmentcrud/service/CityService.java

```diff
@@ -0,0 +1,48 @@
+package likelion.likelionassignmentcrud.service;
+
+import likelion.likelionassignmentcrud.domain.City;
+import likelion.likelionassignmentcrud.domain.Country;
+import likelion.likelionassignmentcrud.dto.city.CityResponseDto;
+import likelion.likelionassignmentcrud.dto.city.CitySaveDto;
+import likelion.likelionassignmentcrud.repository.CityRepository;
+import likelion.likelionassignmentcrud.repository.CountryRepository;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.List;
+
+@Service
+@RequiredArgsConstructor
+public class CityService {
+
+    private final CityRepository cityRepository;
+    private final CountryRepository countryRepository;
+
+    // 도시 저장
+    @Transactional
+    public void saveCity(CitySaveDto citySaveDto) {
+       Country county = countryRepository.findById(citySaveDto.country_id())
+           .orElseThrow(IllegalArgumentException::new);
```

Comment:
이런 식으로 예외를 던지게 되면 메시지가 없어서 디버깅에 어려움을 겪을 수 있어요 😢😢
아래에 작성한 것처럼 예외 메시지를 명확히 보여지도록 작성하면 어떤 값이 잘못되었는지 바로 파악할 수 있어서 유지보수에 큰 도움이 될 거예요-!-!

```suggestion
           .orElseThrow(() -> new IllegalArgumentException("해당 countryId에 해당하는 국가가 없습니다."));
```